include/nbl/builtin/hlsl/type_traits.hlsl

```diff
@@ -294,6 +294,7 @@ struct is_signed : bool_constant<
 
 }
 
+// TODO: struct & trait is named wrong
 template<class T>
 struct is_spirv_type : false_type {};
 template<class T, class Storage>
```

Comment:
TODO for me, detect:
- `is_spirv_type`
- `is_spirv_opaque_type`
- `spirv::is_pointer`
- `spirv::is_bda_pointer`